### PR TITLE
PlanUML diagram of the DB structure

### PR DESCRIPTION
**Pull request by rpkyrych**

### CODE REVIEW

File: documentation/db-structure.puml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+organization o-- contactData: contains
+organization o-- organizationData: contains
+organization o-- mediaData: contains
+organization "1" *-- "many" user : "organisationId"
+organization "1" *-- "many" board : "organisationId"
+board "1" *-- "many" task : "boardId"
+user "1" *-- "many" task: ""
+
+@enduml
```
Comment (Wolshebnik) [inline documentation/db-structure.puml:86]: add space

File: documentation/db-structure.puml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   __v: int32
+}
+
+entity board {
+    _id: objectid
+    name: string
+    organizationId: objectid
+    tasks: list
+    __v: int32
+}
```
Comment (Wolshebnik) [inline documentation/db-structure.puml:68]: where is the connection board with columns?
image

Does the task need to be linked to board Columns?

Comment (rpkyrych) [reply documentation/db-structure.puml:68]: That's the plan, but you can always suggest some other variation

File: documentation/db-structure.puml (hunk 1 above)
Comment (rpkyrych) [reply documentation/db-structure.puml:86]: GitHub doesn't require that anymore

Comment (rpkyrych) [reply documentation/db-structure.puml:86]: That is non-mandatory requirement.

Comment (Wolshebnik) [reply documentation/db-structure.puml:86]: please add a blank line. don't break the project structure,

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Review by Kyslytsia — APPROVED** [review]

**Pull request merged**